src/main/resources/config/application-artemis.yml

```diff
@@ -76,7 +76,6 @@ artemis:
         # Jenkins: The token value you use for the Server Notification Plugin and is stored under the notification-token credential above
         artemis-authentication-token-value: <token>
         build-timeout: 30 # Does cancel jenkins builds after 30 minutes to remove build that get stuck
-
```

Comment:
Maybe add the missing line here that no change will be commited :)

Reply:
Done 🙂